azure-container-apps/ReadMe.md

```diff
@@ -1,337 +1,349 @@
-# Deploy 1Password SCIM Bridge on Azure Container Apps
+# Deploy 1Password SCIM Bridge on Azure Container Apps using the Azure Portal
 
-This deployment example describes how to deploy 1Password SCIM Bridge as an app on Azure's [Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service.
+_Learn how to deploy 1Password SCIM Bridge on the [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service._
 
-The deployment consists of two [containers](https://learn.microsoft.com/en-us/azure/container-apps/containers), the SCIM bridge container and a Redis container and an [ingress](https://learn.microsoft.com/en-us/azure/container-apps/ingress-overview) for the SCIM bridge container.
+This deployment consists of two [containers](https://learn.microsoft.com/en-us/azure/container-apps/containers): One for the SCIM bridge and another for Redis. There's also an [ingress](https://learn.microsoft.com/en-us/azure/container-apps/ingress-overview) for the SCIM bridge container. There are a few benefits to deploying 1Password SCIM Bridge on Azure Container Apps:
 
-## In this folder
+- For standard deployments, the service will host your SCIM bridge for ~$16 USD/month (as of January 2024). Container Apps pricing is variable based on activity, and you can learn more on [Microsoft's pricing page](https://azure.microsoft.com/en-us/pricing/details/container-apps/).
+- You don't need to manage a DNS record. Azure Container Apps automatically provides a unique one for your SCIM bridge domain.
+- Azure Container Apps automatically handles TLS certificate management on your behalf.
+- The SCIM bridge can be deployed directly to Azure from the Portal or via the command line in your local terminal. There is no requirement to clone this repository.
```

Comment:
Small suggestion to highlight that this guide is for the portal deployment and have a link out to the AKB guide for Shell based deployments. Feel free to adjust the wording, as there is something that is not 100% sitting correctly with my suggestion. 

```suggestion
- The SCIM bridge can be deployed directly to Azure from the Portal using this guide or via the Azure Shell or command line tools in your local terminal following our [support guide](https://support.1password.com/scim-deploy-azure/). There is no requirement to clone this repository.
```